### PR TITLE
Fix spelling in documentation

### PR DESCRIPTION
**Pull request by samjwu**
Also skip spell check on API Reference Guide

### CODE REVIEW

**Comment by CongMa13** [comment]
Hi @samjwu , thanks for the fix for spelling issue.  

Any thoughts about the failure of Spelling?

File: .spellcheck.yaml
Comment (saadrahim) [inline .spellcheck.yaml:1]: You should not need yaml file here. Rocm-docs-core should contain this

File: docs/API_Reference_Guide.rst [hunk 1]
```diff
@@ -6,62 +6,61 @@ Introduction
 hiptensor Data Types
 ====================
 
-hiptensorStatus_t
+:code:`hiptensorStatus_t`
```
Comment (neon60) [inline docs/API_Reference_Guide.rst]: I think we should just disable the most of the file from spellcheck. I know why the spellchecker is not ignoring this, but I do not know what words should be checked in this file at all.

File: docs/API_Reference_Guide.rst [hunk 2]
```diff
@@ -6,62 +6,61 @@ Introduction
 hiptensor Data Types
 ====================
```
Comment (neon60) [inline docs/API_Reference_Guide.rst:8]:
```suggestion

.. <!-- spellcheck-disable -->

```

File: docs/API_Reference_Guide.rst [hunk 3]
```diff
@@ -70,93 +69,93 @@ hiptensorContractionPlan_t
...
+:code:`hiptensorLoggerSetMask`
 ----------------------
 
 .. doxygenfunction::  hiptensorLoggerSetMask
 
-hiptensorLoggerForceDisable
+:code:`hiptensorLoggerForceDisable`
 ---------------------------
 
 .. doxygenfunction::  hiptensorLoggerForceDisable
```
Comment (neon60) [inline docs/API_Reference_Guide.rst:163]:
```suggestion
.. doxygenfunction::  hiptensorLoggerForceDisable

.. <!-- spellcheck-enable -->
```

File: docs/API_Reference_Guide.rst [hunk 4]
```diff
@@ -70,93 +69,93 @@ hiptensorContractionPlan_t
...
 .. doxygenfunction::  hiptensorLoggerSetLevel
 
-hiptensorLoggerSetMask
+:code:`hiptensorLoggerSetMask`
 ----------------------
 
 .. doxygenfunction::  hiptensorLoggerSetMask
 
-hiptensorLoggerForceDisable
+:code:`hiptensorLoggerForceDisable`
```
Comment (neon60) [inline docs/API_Reference_Guide.rst]: I would revert these.

Comment (samjwu) [reply docs/API_Reference_Guide.rst]: [Revert "Use code directive to escape code keywords"](https://github.com/ROCmSoftwarePlatform/hipTensor/pull/155/commits/7b8c448f979cd02350184e31ca7a6667d161329d) 
[7b8c448](https://github.com/ROCmSoftwarePlatform/hipTensor/pull/155/commits/7b8c448f979cd02350184e31ca7a6667d161329d)
This reverts commit https://github.com/samjwu/hipTensor/commit/7be7e3446830f2d7dd2760f140c7dfa30a246f78.

File: docs/API_Reference_Guide.rst (hunk 1 above)
Comment (samjwu) [reply docs/API_Reference_Guide.rst]: Sounds good to me

[Disable spellcheck on API Reference Guide](https://github.com/ROCmSoftwarePlatform/hipTensor/pull/155/commits/0d324d9a1692c11df4d650cfbb216fc8e52e7241)

**Review by cgmillette — APPROVED** [review]

**Pull request merged**